### ISSUE TITLE
does it work with multiple monitors?

### ISSUE THREAD

**Issue by jmlucjav** [AlfredoSequeida/hints#3]
Hi,

I quickly set it up and tried it, I get hints only in my current monitor, not other ones.
Ubuntu 24.04

**Comment by AlfredoSequeida** [comment]
Hey welcome, I just tried it on multiple monitors (i3wm) and that is working for me. I know I recently made some changes on gnome because I have seen some things behave different there. I wonder if multi-monitor setups also behave differently on Gnome. I'll give it a try on my end and report back. Thank you for noting this!

**Comment by AlfredoSequeida** [comment]
@jmlucjav I was not able to replicate the issue on a clean installation of Ubuntu 24.04. I tested it on a laptop with an external monitor attached and on both monitors I was able to see hints as expected. I'm wondering if you might have a HiDPI display or special scaling setup? I have noticed issues with hint placement in the past because of this. There is a section about setting a "scaling_factor" in the wiki if this applies to you.

**Comment by jmlucjav** [comment]
thanks for looking into this. Let me investigate on my side...

**Comment by jmlucjav** [comment]
just to be clean, on ubuntu 24.04/x11 the compositing is already there by default, no need to do anything right? Just checking I am not missing some prerequisite...I see weird things when trying on a laptop

**Comment by AlfredoSequeida** [comment]
Yes, it should just work out of the box after an installation. It won't work on the live image though. I'm not sure why that is, but the live environment is a little stange.

**Comment by AlfredoSequeida** [comment]
@jmlucjav  Any update on this?

**Comment by jmlucjav** [comment]
hi, 

it is not working for me on ubuntu 24.04/x11 (native install), with 3 monitors. 

In one monitor I get this (the other monitors get no hints at all):
![image](https://github.com/user-attachments/assets/d5e4a762-ad80-48cc-b63c-1f2f09796a41).

If I launch it in another shell in another monitor, same thing happens, only the terminal (be it kitty, wezterm...) gets hints, in that monitor.

**Comment by AlfredoSequeida** [comment]
Oh, so if I understand, what you're describing is actually the expected behaviour. Hints are drawn only on the focused window. So since you are launching the command from a terminal, that window is that is focused when hints are drawn. This is why the docs mention setting a keyboard shortcut to launch hints on command (with the currently focused window).

So as a test, you can do a `sleep 3; hints`,  then focus on a different window and wait for the 3 seconds to be over to see what happens. The expected outcome is that the focuses window (the window that is currently selected) will show hints.

**Comment by jmlucjav** [comment]
my bad, wanted to just test before puting this in my kanata config etc. Ok it's hooked into a shortcut, and it's working. On some windows the transparency is working and there are other gotchas but I'll keep an eye on open issues and update when available.

Thanks for this!!
